src/main/java/io/kestra/plugin/langchain4j/embeddings/KestraKVStore.java

```diff
@@ -28,11 +29,36 @@
 @Getter
 @SuperBuilder
 @NoArgsConstructor
-@Plugin(beta = true)
 @JsonDeserialize
 @Schema(
     title = "In-memory Embedding Store that then store its serialization form as a Kestra K/V pair"
 )
+@Plugin(
+    beta = true,
+    examples = {
+        @Example(
+            full = true,
+            title = "Ingest documents into a KV embedding store.\\nWARNING: the KV embedding store is for quick prototyping only, as it stores the embedding vectors in a K/V Store and load them all in memory.",
```

Comment:
```suggestion
            title = "Ingest documents into a KV embedding store.\\nWARNING: the `KestraKVStore` embeddings are for quick prototyping only; since they are stored in a KV Store and loaded from there into memory, this won't scale with a large number of documents.",
```
I'd clarify that the limitation is with a large nr of documents